### PR TITLE
Improvements on standalone node execution

### PR DESCRIPTION
**Pull request by IAvecilla**
## What ❔

Documented a guide on setting up and running a local consensus node, along with a new Makefile containing commands for easy execution. Also modifies the database directory path for each node.

## Why ❔

The README and the Makefile would streamline generating the configuration for all nodes with a single command and simplify running the desired node. About the modification for the database directory path, it seems beneficial for running multiple nodes on a single machine, especially for local testing. However, in the broader architecture of the Test Framework, each node is expected to operate within a distinct Docker container or a similar setup. In that context, this change might not be useful.

### CODE REVIEW

File: node/tools/src/main.rs [hunk 1]
```diff
@@ -100,11 +100,13 @@ async fn main() -> anyhow::Result<()> {
...
-    let storage = RocksdbStorage::new(
-        ctx,
-        &configs.executor.genesis_block,
-        Path::new("./database"),
-    );
+    // Creates a database directory for each node, this allows to run multiple nodes on the same machine.
+    let database_path = if let Some((consensus_config, _)) = &configs.consensus {
+        Path::new("database").join(consensus_config.public_addr.to_string())
+    } else {
+        Path::new("database").join(configs.executor.server_addr.to_string())
```
Comment (pompon0) [inline node/tools/src/main.rs:107]: fyi, "database" is a flag now

File: node/tools/Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.PHONY: run-node node-configs clean
+IP?=127.0.0.1:8000
```
Comment (pompon0) [inline node/tools/Makefile:2]: is this a good default? This port has some canonical uses https://www.speedguide.net/port.php?port=8000 , so perhaps something more random would be preferred.

File: node/tools/Makefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+run-node:
+	export RUST_LOG=INFO && cargo run -- --config-file node-configs/${IP}/config.json --node-key node-configs/${IP}/node_key --validator-key node-configs/${IP}/validator_key --database ./database/${IP} 
+
```
Comment (pompon0) [inline node/tools/Makefile:6]: nit: note that if you execute binary from "node-configs/${IP}" working directory, you don't have to set any flags.

File: node/tools/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Running a test node
+
+1. Generate a file named `config.txt` in the root directory of the tools crate, containing node addresses in the format `IP:PORT`, with each address on a separate line.
```
Comment (pompon0) [inline node/tools/README.md:3]: nit: this is a metaconfig in a sense (used to generate configs), so perhaps giving it a different name than "config" would be helpful.

**Review by pompon0 — APPROVED** [review]

File: node/tools/src/main.rs (hunk 1 above)
Comment (pompon0) [reply node/tools/src/main.rs]: that was intentional, thx for updating the comment

File: node/tools/Makefile (hunk 2 above)
Comment (IAvecilla) [reply node/tools/Makefile]: Updated here [9fe5437](https://github.com/matter-labs/era-consensus/pull/55/commits/9fe5437f3bda51f8010a2264f4aa912ca4c1229c)!

File: node/tools/README.md (hunk 4 above)
Comment (IAvecilla) [reply node/tools/README.md]: Sounds reasonable, I'm using `addresses` now to be more specific

**Pull request merged**